build/macos/flash-serial-macos.sh

```diff
@@ -4,7 +4,7 @@ cd -- "$(dirname "$BASH_SOURCE")"
 
 usage() {
     printf "Usage: ${0} keyboard:keymap:target /path/to/serial/device\n";
-    printf "If you need to find your serial device, try running: ls /dev/cu.*"
+    printf "If you need to find your serial device, try running: ls /dev/tty.usbserial*"
```

Comment:
Note, the CH340 is shown as `/dev/tty.usbserial-14510` for me on macOS Mojave (10.14.1) using the system drivers.